app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/InfoPetFragment.java

```diff
@@ -15,31 +18,133 @@
+import org.pesmypetcare.mypetcare.activities.views.CircularImageView;
 import org.pesmypetcare.mypetcare.databinding.FragmentInfoPetBinding;
+import org.pesmypetcare.mypetcare.features.pets.Gender;
+import org.pesmypetcare.mypetcare.features.pets.Pet;
 
 import java.util.Objects;
 
 
 public class InfoPetFragment extends Fragment {
+    private static Drawable petProfileDrawable;
+    private static boolean isImageModified;
+    private static Pet pet = new Pet("Linux");
+
     private FragmentInfoPetBinding binding;
     private Button birthDate;
-    private Boolean modified;
+    private boolean modified;
     private String newWeight;
     private String newName;
     private String newBreed;
     private String newGender;
-
+    private CircularImageView petProfileImage;
+    private InfoPetCommunication communication;
 
     @Override
     public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
-        modified = false;
         binding = FragmentInfoPetBinding.inflate(inflater, container, false);
+        communication = (InfoPetCommunication) getActivity();
+
         updatePetListeners();
         setCalendarPicker();
         setGenderDropdownMenu();
+        setPetProfileImage();
+        initializePetInfo();
+
+        modified = false;
+
         return binding.getRoot();
     }
 
+    /**
+     * Initialize the pet info.
+     */
+    private void initializePetInfo() {
+        if (pet != null) {
+            Objects.requireNonNull(binding.petName.getEditText()).setText(pet.getName());
+            Objects.requireNonNull(binding.breed.getEditText()).setText(pet.getBreed());
+            Objects.requireNonNull(binding.weight.getEditText()).setText(String.valueOf(pet.getWeight()));
+            Objects.requireNonNull(binding.recommendedKcal.getEditText())
+                .setText(String.valueOf(pet.getRecommendedDailyKiloCalories()));
+            Objects.requireNonNull(binding.washFrequency.getEditText())
+                .setText(String.valueOf(pet.getWashFrequency()));
+            binding.inputBirthMonth.setText(pet.getBirthDate());
+
+            setGender();
+            setPathologies();
+        }
+    }
+
+    /**
+     * Sets the pet pathologies if exists.
+     */
+    private void setPathologies() {
+        if (pet.getPathologies() != null) {
+            Objects.requireNonNull(binding.pathologies.getEditText()).setText(pet.getPathologies());
+        }
+    }
+
+    /**
+     * Sets the gender of the pet.
+     */
+    private void setGender() {
+        if (pet.getGender() == Gender.MALE) {
+            binding.inputGender.setText(R.string.male);
+        } else {
+            binding.inputGender.setText(R.string.female);
+        }
+    }
+
+    /**
+     * Sets the pet profile image.
+     */
+    private void setPetProfileImage() {
+       petProfileImage = binding.imgPet;
+
+       if (petProfileDrawable == null) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['if' has incorrect indentation level 7, expected level should be 8.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5223641)